pkg/pillar/cmd/domainmgr/sriov.go

```diff
@@ -0,0 +1,84 @@
+package domainmgr
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/vishvananda/netlink"
+)
+
+func vfCount(ib types.PhysicalIOAdapter) (uint8, error) {
+	v, exists := ib.Cbattr[types.VfCountFieldName]
+
+	if exists {
+		c, err := strconv.Atoi(v)
+		if err != nil {
+			return 0, fmt.Errorf("Cannot convert %s to integer. Value %s, error: %v", types.VfCountFieldName, v, err)
+		} else {
+			return uint8(c), nil
+		}
+	} else {
+		return 0, nil
+	}
+}
+
+func createVF(device string, vfCount uint8) error {
+	f := fmt.Sprint(types.NicLinuxPath, device, types.NumvfsDevicePath)
+	err := os.WriteFile(f, []byte(fmt.Sprint("%n", vfCount)), 0644)
```

Comment:
Itoa in strconv package makes more sense if you are only printing the number,

Reply:
👍